packages/perseus/src/widgets/dropdown/dropdown.tsx

```diff
@@ -80,31 +86,58 @@ class Dropdown extends React.Component<Props> implements Widget {
         ];
 
         return (
-            <div
-                // NOTE(jared): These are required to prevent weird behavior
-                // When there's a dropdown in a zoomable table.
-                onClick={(e) => {
-                    e.stopPropagation();
-                }}
-                onTouchStart={(e) => {
-                    e.stopPropagation();
-                }}
-            >
-                <SingleSelect
-                    placeholder=""
-                    onChange={(value) => this._handleChange(parseInt(value))}
-                    selectedValue={String(this.props.selected)}
-                    disabled={this.props.apiOptions.readOnly}
-                >
-                    {children}
-                </SingleSelect>
-            </div>
+            <UniqueIDProvider scope="dropdown-widget" mockOnFirstRender={true}>
+                {(ids) => (
+                    <View
+                        // NOTE(jared): These are required to prevent weird behavior
+                        // When there's a dropdown in a zoomable table.
+                        onClick={(e) => {
+                            e.stopPropagation();
+                        }}
+                        onTouchStart={(e) => {
+                            e.stopPropagation();
+                        }}
+                    >
+                        {this.props.visibleLabel && (
+                            <LabelLarge
+                                tag="label"
+                                id={ids.get("dropdown-label")}
+                            >
+                                {this.props.visibleLabel}
+                            </LabelLarge>
+                        )}
+                        <SingleSelect
+                            id={ids.get("dropdown")}
+                            placeholder=""
+                            onChange={(value) =>
+                                this._handleChange(parseInt(value))
+                            }
+                            selectedValue={String(this.props.selected)}
+                            disabled={this.props.apiOptions.readOnly}
+                            aria-label={
+                                this.props.ariaLabel ||
+                                this.context.strings.selectAnAnswer
+                            }
+                            aria-labelledby={ids.get("dropdown-label")}
```

Comment:
I wonder if this should be moved into the `aria-label` logic. In the case where both a visible label and an aria label are specified, the visible label would be read instead of the aria label (since `aria-labelledby` takes precedence). Perhaps `aria-label` could be changed to:
```
aria-label={
    this.props.ariaLabel ||
    this.props.visibleLabel ||
    this.context.strings.selectAnAnswer
}
```